generator/03-syndesis-git-proxy.yml.mustache

```diff
@@ -65,7 +65,12 @@
           readinessProbe:
             tcpSocket:
               port: 8080
-            initialDelaySeconds: 1
+            initialDelaySeconds: 10
```

Comment:
I thought that a first check after a second of container startup is a bit small. But we can leave it. The github proxy has to wait for the keycloak server to come up anyway, so in the beginning it will be killed very short after startup anyway. Maybe its better to not kick in a readiness check during this labor pains ?